front/src/components/organisms/UserProfileResourcesWidget.tsx

```diff
@@ -100,6 +100,8 @@ export const UserProfileResourcesWidget = ({
 
   const [sortOrder, setSortOrder] = useState<TSortOrder>('desc')
   const [isSortByVotesActive, setIsSortByVotesActive] = useState(false)
+  const [filters] = useState<TFilters>({})
```

Comment:
This function here is unnecessary. If you added it just to avoid type requirement complaints, consider to convert filters in optional and omit the prop in this file. 
Remember the issue functionality was only required for Category page (UserProfileResoucesWidget already hides the sort component).

Reply:
Fixed - thank you so much for the feedback! I still had to keep variations of the tests due to VotesDateController's new behavior, but if there is anything else that needs attention, please feel free to let me know!